mitodl/mitxonline
# Add migration to deactivate program enrollments

### What are the relevant tickets?
Fix https://github.com/mitodl/hq/issues/3780

### Description (What does it do?)
Add migration to deactivate program enrollments



### How can this be tested?
Create an enrollment in a program with readable_id = program-v1:MITxT+18.01x.
Then run the migration and make sure that your enrollment was deactivated.
